ContinuousML/NQ_DR_4_10_20_Ideal_27/2_Training_Workflow/dvc.yaml

```diff
@@ -0,0 +1,9 @@
+stages:
+  training:
+    cmd: python ../../../src/Model_Training/Model_Training.py
+    deps:
+    - ../../../src/Model_Training/Model_Training.py
+    - ../1_Label_And_Feature_Workflow/Full_Features.csv
```

Comment:
- Replace Full_features.csv with Reduced_Features.csv
- Reduced_Features.csv can be produced from Full_features with fcolumn filtering Provided by Selected_features.csv